CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Fixed
+
+- Fix badly named nginx location from `export` to `restricted`
```

Comment:
You have to refer to the app you are fixing. `edxapp` is one application among others.

Reply:
I do refer to nginx, here and in commit message.

Reply:
The `nginx` service from the `edxapp` application.